docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -173,6 +173,195 @@ It is also possible to configure the scope values that would be used via the sco
 
 NOTE: If both the deployment and subsystem configuration is present, then the subsystem configuration will override the deployment values.
 
+== Sending Request Objects as a JWT
+
+NOTE: Since this attribute is a preview level feature, the WildFly server's stability level must be set to `preview`
+or lower to enable the ability to add `request` or `request_uri` to authentication requests. The stability level must
+not be changed to community or higher after configuring the attributes related to authentication request format. For more
+information about the stability levels of the WildFly server, please refer to the
+link:Admin_Guide{outfilesuffix}[Admin Guide], particularly the
+link:Admin_Guide{outfilesuffix}#Feature_stability_levels[Feature stability levels] section.
+
+According to the https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests[OpenID Connect specifications],
+the `request` and the `request_uri` parameters enable Authentication Requests to be signed and optionally encrypted.
+Both of these parameters are optional and requests sent using these parameters are represented as a JWT. The
+`elytron-oidc-client` subsystem allows users to specify if they want to send their authentication requests as a JWT using
+the `autnentication-request-format` attribute. The `authentication-request-format`, `request-object-signing-algorithm`,
```

Comment:
s/autnentication/authentication

Reply:
Might be good to move the last sentence here that talks about all the attributes to somewhere after the attributes have been introduced.